test/integration/BUILD

```diff
@@ -9,6 +9,14 @@ licenses(["notice"])  # Apache 2
 
 envoy_package()
 
+py_test(
+    name = "test_nighthawk_test_server",
```

Comment:
I wonder if tests for our test-code should go into a distinct place for clarity (arguably it's not test-code as we offer it for production purposes too, but still)

Reply:
Makes sense, how about we create a subdirectory like `integration/unit_tests` ? Open to suggestions.

Reply:
Addressed in 6cd4657596aa0a0d383bfd63d65995a30c9aa84a.